app/src/main/java/ch/epfl/sdp/blindly/user/User.kt

```diff
@@ -76,7 +76,7 @@ class User private constructor(
          *
          * @param location the location of the User
          */
-        fun setLocation(location: String) = apply {
+        fun setLocation(location: List<Double>) = apply {
```

Comment:
Maybe you should add checks on the input length (that it only contains two doubles)